app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  after_save :update_post_counter
+
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_post_counter
```

Comment:
- [ ] OPTIONAL
Following best practices `update_post_counter` should be marked as private after implementing the `after_save` method.

Reply:
Noted...Thank you